sensor_msgs/Image img_r
sensor_msgs/Image img_g
sensor_msgs/Image img_b
sensor_msgs/Image img_nir
sensor_msgs/Image img_swir
